controller/label_blackbox_test.go

```diff
@@ -119,10 +119,10 @@ func (rest *TestLabelREST) TestUpdate() {
 				Type:    label.APIStringTypeLabels,
 			},
 		}
-		_, updated := test.UpdateLabelOK(t, svc.Context, svc, ctrl, testFxt.Spaces[0].ID, testFxt.Labels[0].ID, &payload)
+		resp, updated := test.UpdateLabelOK(t, svc.Context, svc, ctrl, testFxt.Spaces[0].ID, testFxt.Labels[0].ID, &payload)
 		assert.Equal(t, newName, *updated.Data.Attributes.Name)
-		assert.False(t, updated.Data.Attributes.UpdatedAt.After(time.Now()), "Label was not updated, UpdatedAt after Now()")
-		compareWithGoldenUUIDAgnostic(t, filepath.Join(rest.testDir, "update", "ok_update_label.golden.json"), updated)
+		compareWithGoldenUUIDAgnostic(t, filepath.Join(rest.testDir, "update", "ok.label.golden.json"), updated)
+		compareWithGoldenUUIDAgnostic(t, filepath.Join(rest.testDir, "update", "ok.headers.golden.json"), resp)
```

Comment:
I think I've used `resp.Headers()` instead of just `resp`. Don't know if your's works.